packages/react-components/src/components/Skeleton/SkeletonPlaceholder.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+
+import cx from 'clsx';
+
+import styles from './Skeleton.module.scss';
+
+const baseClass = 'skeleton-placeholder';
+
+export interface ISkeletonPlaceholder {
+  /**
+   * Specify if the element is square shape
+   */
+  square?: boolean;
+  /**
+   * Specify the size of the element (wdith and height in px)
+   */
+  size?: number;
+  /**
+   * Specify if the animation is enabled
+   */
+  animated?: boolean;
+}
+
+export const SkeletonPlaceholder: React.FC<ISkeletonPlaceholder> = ({
```

Comment:
This component is supposed to mock the avatar component? Maybe it should contain the word 'Avatar' in its name?